src/rdkafka_conf.c

```diff
@@ -1013,6 +1013,38 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "the client's OAUTHBEARER token.",
           _UNSUPPORTED_OAUTHBEARER
         },
+        { _RK_GLOBAL, "sasl.oauthbearer.method", _RK_C_STR,
```

Comment:
This should probably be an s2i with enums in rdkafka_conf.h for the two methods:
`RD_KAFKA_SASL_OAUTHBEARER_METHOD_DEFAULT` and `.._OIDC`  which maps to "default" and "oidc" in the s2i list. See other s2i properties how this is done. This also changes the method field in conf_t to that enum type.